pkg/ddl/modify_column.go

```diff
@@ -1248,7 +1676,7 @@ func GetModifiableColumnJob(
 }
 
 // needChangeColumnData is a strong check to decide whether we need to change the column data.
-// If it returns true, it means we don't need the reorg no matter what the data is..
+// If it returns true, it means we don't need the reorg no matter what the data is.
```

Comment:
This function return `true` means don't need reorg, it is confused with the function name  `needChangeColumnData`